run/docker/docker-compose.yml

```diff
@@ -24,25 +24,26 @@ services:
       - DEFAULT_GROUPS=prime256v1:secp384r1:frodo640aes:frodo640shake:frodo976aes:frodo976shake:frodo1344aes:frodo1344shake:kyber512:p256_kyber512:kyber768:p384_kyber768:x25519_kyber768:kyber1024:bikel1:bikel3:bikel5:hqc128:hqc192:hqc256
   
   curl:
-    image: qujata/curl:1.0.0
+    image: qujata/curl:1.1.0
     container_name: qujata-curl
     ports:
       - 3010:3010
     environment:
       - DEFAULT_GROUPS=prime256v1:secp384r1:frodo640aes:frodo640shake:frodo976aes:frodo976shake:frodo1344aes:frodo1344shake:kyber512:p256_kyber512:kyber768:p384_kyber768:x25519_kyber768:kyber1024:bikel1:bikel3:bikel5:hqc128:hqc192:hqc256
 
   api:
-    image: qujata/api:1.0.0
+    image: qujata/api:1.1.0
     container_name: qujata-api
     ports:
       - 3020:3020
     environment:
       - CURL_URL=http://qujata-curl:3010
       - PLATFORM_EXPORTER_URL=http://qujata-platform-exporter:5000
+      - ITERATIONS_OPTIONS=100:500:1000:2000:5000:10000:50000
       - DEFAULT_GROUPS=prime256v1:secp384r1:frodo640aes:frodo640shake:frodo976aes:frodo976shake:frodo1344aes:frodo1344shake:kyber512:p256_kyber512:kyber768:p384_kyber768:x25519_kyber768:kyber1024:bikel1:bikel3:bikel5:hqc128:hqc192:hqc256
 
   portal:
-    image: qujata/portal:1.0.0
+    image: qujata/portal:1.1.0
```

Comment:
add the new version to kuberenets charts as well
also, push all other images to 1.1.0 (except platform-exporter)

Reply:
fixed